src/components/TextField/README.md

```diff
@@ -211,25 +202,16 @@ class TextFieldExample extends React.Component {
 Use when input text should be a number.
 
 ```jsx
-class NumberFieldExample extends React.Component {
-  state = {
-    value: '1',
-  };
-
-  handleChange = (value) => {
-    this.setState({value});
-  };
-
-  render() {
-    return (
-      <TextField
-        label="Quantity"
-        type="number"
-        value={this.state.value}
-        onChange={this.handleChange}
-      />
-    );
-  }
+function NumberFieldExample() {
+  const [value, setValue] = useState('1');
+  return (
+    <TextField
+      label="Quantity"
+      type="number"
+      value={value}
+      onChange={(newValue) => setValue(newValue)}
```

Comment:
Can this (and all the below onChange usages) be `onChange={setValue}`?

Reply:
That would mean react receives all arguments which unfortunately triggers an error 😞